### PR TITLE
Survey link

### PR DESCRIPTION
**Pull request by Pantkowsky**
Added the survey link in blue colour underneath the TIP. Fixes https://github.com/corradio/electricitymap/issues/113

### CODE REVIEW

File: api/static/index.html [hunk 1]
```diff
@@ -100,11 +100,15 @@
                 The <span style="color: lightgray">blinking arrows</span> represent electricity <span style="color: lightgray">imports and exports</span>.<br />
                 <br />
                 <span style="color: lightgray; font-size: 14px; color: white;"><i>Tip: Click on a country to start exploring ⟶</i></span><br />
+		<br />
                 <div class="country-picker-container" style="text-align: center">
                     <select class="country-picker" onchange="onSelectedCountryChange();"></select>
                 </div>
             </div>
-            
+	    <svg class="country-table" style="display: none"></svg>
```
Comment (corradio) [inline api/static/index.html]: Something seems wrong here. Why is this `svg` tag added?

File: api/static/index.html [hunk 2]
```diff
@@ -100,11 +100,15 @@
                 The <span style="color: lightgray">blinking arrows</span> represent electricity <span style="color: lightgray">imports and exports</span>.<br />
                 <br />
                 <span style="color: lightgray; font-size: 14px; color: white;"><i>Tip: Click on a country to start exploring ⟶</i></span><br />
+		<br />
```
Comment (corradio) [inline api/static/index.html]: Indentation

File: api/static/index.html [hunk 3]
```diff
@@ -100,11 +100,15 @@
...
                 <div class="country-picker-container" style="text-align: center">
                     <select class="country-picker" onchange="onSelectedCountryChange();"></select>
                 </div>
             </div>
-            
+	    <svg class="country-table" style="display: none"></svg>
+	    <div class="country-table-initial-text">
+		<a href="https://docs.google.com/forms/d/e/1FAIpQLSc-_sRr3mmhe0bifigGxfAzgh97-pJFcwpwWZGLFc6vvu8laA/viewform?c=0&w=1">We would <strong>love</strong> your feedback. Help us out!</a>
+            </div>
```
Comment (corradio) [inline api/static/index.html]: Indentation

File: api/static/index.html [hunk 4]
```diff
@@ -100,11 +100,15 @@
...
                 <span style="color: lightgray; font-size: 14px; color: white;"><i>Tip: Click on a country to start exploring ⟶</i></span><br />
+		<br />
                 <div class="country-picker-container" style="text-align: center">
                     <select class="country-picker" onchange="onSelectedCountryChange();"></select>
                 </div>
             </div>
-            
+	    <svg class="country-table" style="display: none"></svg>
+	    <div class="country-table-initial-text">
+		<a href="https://docs.google.com/forms/d/e/1FAIpQLSc-_sRr3mmhe0bifigGxfAzgh97-pJFcwpwWZGLFc6vvu8laA/viewform?c=0&w=1">We would <strong>love</strong> your feedback. Help us out!</a>
```
Comment (corradio) [inline api/static/index.html]: You probably want to add `target="_blank"` to open in a new window

File: docker-compose.yml [hunk 5]
```diff
@@ -16,7 +16,7 @@ services:
   feeder:
     build: feeder
     depends_on: [mongo]
-    environment:
+    environment:
```
Comment (corradio) [inline docker-compose.yml]: Unnecessary change

File: feeder/parsers/PL.py [hunk 6]
```diff
@@ -6,4 +6,4 @@
 def fetch_PL(session=None):
     return fetch_ENTSOE(ENTSOE_DOMAIN, COUNTRY_CODE, session)
 
-if __name__ == '__main__': print(fetch_PL())
```
Comment (corradio) [inline feeder/parsers/PL.py]: Unnecessary change

File: api/static/index.html (hunk 1 above)
Comment (Pantkowsky) [inline api/static/index.html]: That is not supposed to be there. But actually, I wanted to include the same line that separates the heading from table content at the bottom, after the TIP, for beter visuals. I found that the <svg> was for that but it does not seem to work inside a table. A line (<hr>) on the other hand is too thick and looks different than the one on top so its not ok to include that. Should there be any line at all though?

File: api/static/index.html [hunk 7]
```diff
@@ -100,11 +100,14 @@
                 The <span style="color: lightgray">blinking arrows</span> represent electricity <span style="color: lightgray">imports and exports</span>.<br />
                 <br />
                 <span style="color: lightgray; font-size: 14px; color: white;"><i>Tip: Click on a country to start exploring ⟶</i></span><br />
+		 <br />
```
Comment (corradio) [inline api/static/index.html]: Please move the `<br />` inside your `div`

File: api/static/index.html [hunk 8]
```diff
@@ -100,11 +100,14 @@
...
                 <div class="country-picker-container" style="text-align: center">
                     <select class="country-picker" onchange="onSelectedCountryChange();"></select>
                 </div>
             </div>
-            
+	    <div class="country-table-initial-text">
```
Comment (corradio) [inline api/static/index.html]: why `class="country-tableinitial-text"`?

File: feeder/parsers/PL.py (hunk 6 above)
Comment (corradio) [inline feeder/parsers/PL.py]: You removed the last (empty) line actually

File: feeder/parsers/ENTSOE.py [hunk 9]
```diff
@@ -40,7 +40,7 @@ def query_production(psr_type, in_domain, session):
     }
     response = session.get(ENTSOE_ENDPOINT, params=params)
     if response.ok: return response.text
-    else: 
+    else:
```
Comment (corradio) [inline feeder/parsers/ENTSOE.py]: Can you make sure no changes are done in this file?

File: feeder/parsers/SI.py [hunk 10]
```diff
@@ -0,0 +1,9 @@
+from ENTSOE import fetch_ENTSOE
+
```
Comment (corradio) [inline feeder/parsers/SI.py]: This file should not be here

File: feeder/parsers/BG.py [hunk 11]
```diff
@@ -0,0 +1,9 @@
+from ENTSOE import fetch_ENTSOE
+
+COUNTRY_CODE = 'BG'
+ENTSOE_DOMAIN = '10YCA-BULGARIA-R'
```
Comment (corradio) [inline feeder/parsers/BG.py]: Add in `feeder.py` instead of having a parser.
Also, make sure the add all the exchanges with neighboring countries
